src/store/slices/node/actionsAsync.ts

```diff
@@ -317,11 +615,31 @@ const removeAliasThunk = createAsyncThunk(
       }
     }
   },
+  { condition: (_payload, { getState }) => {
+    const isFetching = getState().node.aliases.isFetching;
+    console.log('aliases isFetching:', isFetching);
+    if (isFetching) {
+      console.log('Cancelling removeAlias request');
+      return false;
+    }
+  } },
 );
 
-const closeChannelThunk = createAsyncThunk(
+const closeChannelThunk = createAsyncThunk<
+  | {
+      channelStatus: string;
+      receipt?: string | undefined;
+    }
+  | undefined,
+  CloseChannelPayloadType,
+  { state: RootState }
+>(
   'node/closeChannel',
-  async (payload: CloseChannelPayloadType, { rejectWithValue }) => {
+  async (payload: CloseChannelPayloadType, {
+    rejectWithValue,
+    dispatch,
+  }) => {
+    dispatch(setChannelsFetching(true));
```

Comment:
This function can take time, if we set all channels to fetching then I won't be able to fund or get channels while this is happening. Makes all of these interactions exclusive. Not sure if we should separate this or if this makes sense.

cc @mjadach-iv

Reply:
This same pattern happens for tickets

Reply:
We added two new states: `closeChannel` and `redeemTickets` to aid this cases and handle them specifically